src/utils/vacation-data-utils.tsx

```diff
@@ -0,0 +1,42 @@
+import { DailyEntry } from "generated/client";
+import { VacationDayData, VacationWeekData } from "types";
+import getISOWeek from "date-fns/getISOWeek";
+
+/**
+ * Utility class for vacation days
+ */
+export default class VacationDataUtils {
```

Comment:
Convert this to namespace unless you really need static functions.

Reply:
Done